schematic/models/validate_attribute.py

```diff
@@ -1,9 +1,11 @@
+import builtins
 import logging
 import re
 from time import perf_counter
 
 # allows specifying explicit variable types
-from typing import Optional, Union, Tuple
+from typing import Any, Dict, List, Optional, Text, Literal, Union, Tuple
```

Comment:
You don't need to import `Dict` `List` or `Tuple` anymore, you cna just use the lower case verisosn as types `list`, `dict` `tuple`